src/test/java/org/reaktivity/specification/http/internal/FunctionsTest.java

```diff
@@ -48,4 +50,18 @@ public void shouldLoadFunctions() throws Exception
         System.out.println(randomBytes);
     }
 
+    @Test
+    public void headerTest()
+    {
+        byte[] expected = new byte[] { (byte) 0x00, (byte) "name".length(), (byte) 'n', (byte) 'a', (byte) 'm', (byte) 'e',
+                (byte) "value".length(), (byte) 'v', (byte) 'a', (byte) 'l', (byte) 'u', (byte) 'e' };
+        byte[] actual = header("name", "value");
+        Assert.assertArrayEquals(expected, actual);
+
+        byte[] expected2 = new byte[] { (byte) 0x00, (byte) ":status".length(), (byte) ':', (byte) 's', (byte) 't', (byte) 'a',
+                (byte) 't', (byte) 'u', (byte) 's', (byte) "200".length(), (byte) '2', (byte) '0', (byte) '0' };
+        byte[] actual2 = header(":status", "200");
+        Assert.assertArrayEquals(expected2, actual2);
```

Comment:
Wrap `byte[]` returned from EL function as `DirectBuffer`, then overlay `HttpHeaderFW` flyweight generated from `http.idl`, then `assertEquals` on the `header.representation()`, `header.name().asString()` and `header.value().asString()`.